tests/integration-tests/tests/performance_tests/test_starccm/test_starccm/starccm.install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
It's a good practice to describe the goal and usage of a bash script with an initial comment